src/Util/Strings.php

```diff
@@ -396,7 +396,7 @@ private static function linkRegEx(): string
 		return 'https?://                   # http or https protocol
   (?:
 	[^/\s\xA0`!()\[\]{};:\'",<>?«»“”‘’.]    # Domain can\'t start with a .
-	[^/\s\xA0`!()\[\]{};:\'",<>?«»“”‘’]+    # Domain can\'t end with a .
+	[^/\s\xA0`!()\[\]{};:\'",<>?«»“”‘’]*    # Domain can\'t end with a .
```

Comment:
Actually, upon looking at it more, the comment says a domain can't end on a dot but it seems to me with that regex it could? I'm not sure why the two almost identical lines couldn't be joined into one like this, but I assume I'm missing something:
```
[^/\s\xA0`!()\[\]{};:\'",<>?«»“”‘’.]+    # Domain can\'t contain a .
```